src/swcio.hpp

```diff
@@ -30,384 +34,213 @@ class swc_record {
         custom
     };
 
+    kind type = kind::undefined; // record type
```

Comment:
You have changed from member variables with trailing underscore to no underscore.